src/model_scene.hpp

```diff
@@ -32,9 +32,15 @@ struct Material {
 
    bool reference_in_option_file = false;
 
-   auto operator==(const Material&) const {
-      return false;
+//FIX MESSY: Still don't understand default, and I just can't get
+//it to compile.
+#ifdef _WIN32
+   bool operator==(const Material&) const = default;
```

Comment:
For the life of me I couldn't get this to compile, much less fully understand the purpose